common/encfstools.py

```diff
@@ -306,7 +311,7 @@ def mount(self, *args, **kwargs):
                 tmp_kwargs = self.split_kwargs('encfs_reverse')
                 tmp_kwargs['path'] = src
                 tmp_kwargs['config_path'] = src
-                tmp_mount = EncFS_mount(*self.args, symlink = False, **tmp_kwargs)
+                tmp_mount = EncFS_mount(*self.args, symlink = False, read_only = self.read_only, **tmp_kwargs)
```

Comment:
Not necessary here neither. This is just to create a new encfs-config which will be unmounted immediately.